test/controllers/notifications_controller_test.rb

```diff
@@ -12,8 +12,8 @@ class NotificationsControllerTest < ActionController::TestCase
   end
 
   test "should assign unapproved sign slides" do
-    users(:two).signs << signs(:one)
-    signs(:one).slides << slides(:two)
+    users(:two).signs << signs(:default)
```

Comment:
There's something somewhere other than that users.yml that makes them different from sign_owner, super_admin and non_sign_owner because server_permissions_test.rb, emergency_permissions_test.rb and users_permission_test.rb fail with anything other than one/two/james/ross. I haven't been able to find where the difference is.